frontend/src/app/map/map.component.ts

```diff
@@ -213,19 +213,14 @@ export class MapComponent implements OnInit, MapInteraction {
    *  according to biotic assignment data.
    */
   private updateAssignedStationSelection() {
-    let bioticAssignments: BioticAssignment[] = [];
-    if (this.ps.iaMode == 'bioticAssignment' && this.pds.bioticAssignmentData != null && this.pds.selectedPSU != null) {
-      bioticAssignments = this.pds.bioticAssignmentData.BioticAssignment.filter(ba => ba.PSU == this.pds.selectedPSU);
-    }
-
     this.map
       .getLayers()
       .getArray()
       .filter(l => l.get('layerType') == 'station')
       .map(l => <VectorSource>(<Layer>l).getSource())
       .forEach(s =>
         s.getFeatures().forEach(f => {
-          MapSetup.updateStationSelection(f, this.pds);
+          MapSetup.updateAssignedStationSelection(f, this.pds);
```

Comment:
Bugfix: This was forgotten to change to update_assigned_stationselection when refactoring